docs/plugins/transform-react-constant-elements.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: docs
 title: React constant elements transformer
-description:
+description: Treat React JSX elements as value types and hoist them to the highest scope.
```

Comment:
Wasn't this (and the other react plugin descriptions) already done in https://github.com/babel/babel.github.io/pull/1188?

Might want to rebase against master.